index.ts

```diff
@@ -78,7 +82,7 @@ class Money {
         Object.freeze(this)
     }
 
-    static fromInteger(amount: number|any, currency?: string): Money {
```

Comment:
It's a "minor" breaking change, but we're still below 1.x.x, and I bumped versions. This change it might seem too radical 2 years ago, but today most TypeScript code bases are much stricter than before, and this can help developers to spot bugs at "build time", instead of runtime.

Reply:
No controversy at all, I think it's a very welcomed change!